draft-kazuho-httpbis-priority.md

```diff
@@ -210,20 +204,23 @@ there is not a clean mapping to HTTP/3.
 The extensible priority scheme is negotiated using the described mechanism. It is
 identified by the 8-bit value of 2.
 
+Endpoints using this scheme SHOULD emit the Priority header field regardless of
+the result of the negotiation, as that negotiation is a hop-by-hop agreement,
+whereas the Priority header field is an end-to-end signal that might have
+meaningful effect to other nodes that handle the HTTP message.
+
 An intermediary connecting to a backend server SHOULD declare support for the
 extensible priority scheme when and only when all the requests that are to be
 sent on that backend connection originates from one client-side connection that
 has negotiated the use of the extensible priority scheme (see {{fairness}}).
 
-# The Priority HTTP Header Field
+# Urgency Prioritization
 
-The Priority HTTP header field can appear in requests and responses. A client
-uses it to specify the priority of the response. A server uses it to inform
-the client that the priority was overwritten. An intermediary can use the
-Priority information from client requests and server responses to correct or
-amend the precedence to suit it (see {{merging}}).
+Urgency based prioritization uses two fields, `urgency` and `progressive` to
```

Comment:
I think you might want to change "fields" to "parameters", as that is the term we use throughout.

And it might be a good idea to change the section title to indicate that the section discusses various parameters collectively, though I do not have a good suggestion.

Reply:
Thanks, I changed fields to parameters.

The intent was to discuss this as a priority scheme, not just a list of parameters, but maybe there's a better way to do it.